api/sse/tasks.py

```diff
@@ -27,6 +39,37 @@ def send_environment_update_message(environment_key: str, updated_at):
     response.raise_for_status()
 
 
+if settings.AWS_SSE_LOGS_BUCKET_NAME:
+
+    @register_recurring_task(
+        run_every=timedelta(seconds=60),
+    )
+    def update_sse_usage():
+        with influxdb_client.write_api(
+            write_options=WriteOptions(batch_size=1000, flush_interval=2000)
+        ) as write_api:
+            for log in sse_service.stream_access_logs():
+                environment = Environment.get_from_cache(log.api_key)
```

Comment:
I wonder if we'd be better off just grabbing all the (active) environments from the DB and storing in a dictionary in this task? Maybe that's a memory concern though.

Reply:
I have reduced the query count to 1 here, but I have removed the warning for invalid keys, do we care about the warning?